boards/arm/stm32_min_dev/support/openocd.cfg

```diff
@@ -0,0 +1,17 @@
+source [find interface/stlink-v2.cfg]
```

Comment:
Yes, it was working (checked again now). I am able to start the debugger session in my setup here. Is there something more specific that you want me to try? I can step through and continue without any issues, but beyond that I haven't tried much.